pymc/ode/ode.py

```diff
@@ -64,6 +64,9 @@ def odefunc(y, t, p):
         times = np.arange(0.5, 5, 0.5)
 
         ode_model = DifferentialEquation(func=odefunc, times=times, n_states=1, n_theta=1, t0=0)
+
+    Due to the nature of the model (as well as included solvers), the process of ODE solution may perform slowly.
+    A faster alternative library based on PyMC--sunode--has implemented Adams' method and BDF (backward differentation formula).  More information about sunode is available at: https://github.com/aseyboldt/sunode.
```

Comment:
Could you move this further up, to the 2nd line of the docstring? Otherwise it'll end up inside the "Examples" section

Reply:
Change incorporated